src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -247,6 +248,19 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('days_retained', type=int,
                    help='Number of days that soft-deleted blob will be retained. Must be in range [1,365].')
 
+    with self.argument_context('storage blob service-properties update', min_api='2018-03-28') as c:
+        c.argument('delete_retention', arg_type=get_three_state_flag(), arg_group='Soft Delete',
+                   help='Enables soft-delete.')
+        c.argument('days_retained', type=int, arg_group='Soft Delete',
+                   help='Number of days that soft-deleted blob will be retained. Must be in range [1,365].')
+        c.argument('static_website', arg_group='Static Website', arg_type=get_three_state_flag(),
+                   help='Enables static-website.')
+        c.argument('index_document', help='Represents the name of the index document. This is commonly "index.html".',
+                   arg_group='Static Website')
```

Comment:
Is `index.html` a default?

Reply:
default is a null. People are familiar with index.html though.